charts/newrelic-pixie/templates/secret.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if (include "newrelic-pixie.areValuesValid" .) }}
+{{- $licenseKey := include "newrelic-pixie.licenseKey" . -}}
+{{- $apiKey := include "newrelic-pixie.apiKey" . -}}
+apiVersion: v1
```

Comment:
I'd add an if above so we don't generate an empty secret if neither `$licenseKey` nor `$apiKey` are set.

Reply:
`{{- if (include "newrelic-pixie.areValuesValid" .) }}` is checking for license key and api key. 👍

Reply:
But that will evaluate to `true` if `customSecretName` is used, right? In that case I don't see any point to render this secret.

Reply:
Oh yes! I missed this. Fixing it.